kotlin/workflow-runtime/src/main/java/com/squareup/workflow/internal/RealRenderContext.kt

```diff
@@ -27,8 +27,10 @@ import kotlinx.coroutines.CompletableDeferred
 
 /**
  * An implementation of [RenderContext] that builds a [Behavior] via [buildBehavior].
+ *
+ * Not for general application use.
  */
-internal class RealRenderContext<StateT, OutputT : Any>(
+class RealRenderContext<StateT, OutputT : Any>(
```

Comment:
Very sad to make this public. See above comment.

Reply:
I brought this up in the Kotlin Slack. I should file a ticket: https://kotlinlang.slack.com/archives/C0B8PUJGZ/p1555962527006200

Reply:
When you do, lemme know I will vote.